web/playground/src/examples.js

```diff
@@ -1,28 +1,31 @@
 const examples = {
   "introduction.prql": [
     "sql",
-    `from invoices
-filter invoice_date >= @1970-01-16
-derive {                        # This adds columns
-  transaction_fees = 0.8,
-  income = total - transaction_fees  # Columns can use other columns
-}
-filter income > 1     # Transforms can be repeated.
-group customer_id (   # Use a nested pipeline on each group
-  aggregate {         # Aggregate each group to a single row
-    average total,
-    sum_income = sum income,
-    ct = count sum_income,
+    `from invoices                        # PRQL query begins with a table
+derive {                             # "derive" adds columns
+  transaction_fees = 0.8,            # "=" sets a column name
+  income = total - transaction_fees  # Calculations can use other column names
+}                                    # "#" starts a comment - even at start of line
+filter income > 1                    # "filter" replaces both of SQL's WHERE & HAVING
+filter invoice_date >= @1970-01-16   # Clear date syntax
+group customer_id (                  # "group" performs the pipeline in (...) on each group
+  aggregate {                        # "aggregate" reduces each group to a single value
+    average total,                   # get the average of the total for each customer_id
+    sum_income = sum income,         # ... and the total of income
+    ct = count customer_id,          # ... and the count of items
   }
 )
-sort {-sum_income}    # Decreasing order
-take 10               # Limit to top 10 spenders
-join c=customers (==customer_id)
-derive name = f"{c.last_name}, {c.first_name}"
-select {              # Select only these columns
-  c.customer_id, name, sum_income
-}
-derive db_version = s"version()" # S-string, escape hatch to SQL
+join c=customers (==customer_id)     # join on "customer_id" from both tables
+derive name = f"{c.last_name}, {c.first_name}" # F-strings like Python
+derive db_version = s"version()"     # S-string give escape hatch to SQL
+select {                             # Select certain columns
+  c.customer_id, name, sum_income, ct, db_version,
+}                                    # trailing comma ignored
```

Comment:
Should we line break these and have the comment at the end? At the moment the comment is on a different line (no travesty though!)

Reply:
I think I like an example of select { ... } all on one line.